lib/flutter_svg_provider.dart

```diff
@@ -70,7 +77,7 @@ class Svg extends ImageProvider<SvgImageKey> {
       ),
       clipToViewBox: false,
       colorFilter:
-          ColorFilter.mode(key.color ?? Colors.transparent, BlendMode.srcATop),
+      ColorFilter.mode(key.color ?? Colors.transparent, BlendMode.srcATop),
```

Comment:
formart

Reply:
```dart
-      ColorFilter.mode(key.color ?? Colors.transparent, BlendMode.srcATop),
+          ColorFilter.mode(key.color ?? Colors.transparent, BlendMode.srcATop),
```

Reply:
Here I think the tabbed one was the correct one because it is the value of colorFilter, isn't it?

Reply:
Sorry! I've read it wrong. It's my first pull request where I've gotten annotations! Haha!